internal/backend/sftp/sftp.go

```diff
@@ -226,11 +226,19 @@ func buildSSHCommand(cfg Config) (cmd string, args []string, err error) {
 		args = append(args, "-p", port)
 	}
 	if cfg.User != "" {
-		args = append(args, "-l")
-		args = append(args, cfg.User)
+		args = append(args, "-l", cfg.User)
 	}
-	args = append(args, "-s")
-	args = append(args, "sftp")
+
+	if cfg.Args != "" {
```

Comment:
Please extend `if cfg.Command != "" {` to return an error if both the command and args parameters are set.